README.md

```diff
@@ -0,0 +1,33 @@
+## TWOMICE.ME
+
+A very little potentially useful information about Allen Shaw, known on Drupal.org and other forums as TwoMice.
+
+
+## CURRENT POSITION 
```

Comment:
Please use Title Case in place of UPPERCASE headings like this one.

Reply:
Done